xbmc/interfaces/legacy/ModuleXbmcplugin.h

```diff
@@ -438,6 +439,33 @@ namespace XBMCAddon
     void setProperty(int handle, const char* key, const String& value);
 #endif
 
+#ifdef DOXYGEN_SHOULD_USE_THIS
+    ///
+    /// \ingroup python_xbmcplugin
+    /// @brief \python_func{ xbmcplugin.getWatchedMode(handle, content) }
+    ///-------------------------------------------------------------------------
+    /// Get the watched mode for given content type:
+    ///  WatchedModeAll, WatchedModeUnwatched, or WatchedModeWatched.
+    ///
+    /// @param handle      integer - handle the plugin was started with.
+    /// @param content     [opt] string - content type; defaults to plugin content type.
+    ///
+    ///
+    /// ------------------------------------------------------------------------
+    ///
+    /// **Example:**
+    /// ~~~~~~~~~~~~~{.py}
+    /// ..
+    /// xbmcplugin.getWatchedMode(int(sys.argv[1]), 'tvshows')
+    /// ..
+    /// ~~~~~~~~~~~~~
+    ///
+    getWatchedMode(...);
+    ///@}
+#else
+    int getWatchedMode(int handle, const char* content = NULL);
```

Comment:
if you mean directory IMO `xbmcplugin.getWatchedMode` is not the best method name, should be something like `xbmcplugin.getDirectoryWatchedMode`